docs_src/src/pages/documentation/example_app/index.mdx

```diff
@@ -65,3 +78,87 @@ And he was done! The Robyn CLI created a new application with the following stru
   />
 </div>
 
+If you choose to go with the structured scaffold, this is how your project will look like
+
+> note: at the moment, only no-db and sqlalchemy are supported here, you can always plug in other integrations as you see fit
```

Comment:
why not the other ones? we support postgres/prisma/sqlalchemy/sqlite in the simple one

Reply:
prisma/nosql we can add in the future...

sqlite, MySQL, oracle are supported by sqlalchemy... maybe we can add commented envs and requirements

Reply:
added

Reply:
> prisma/nosql we can add in the future...

let's ship a feature complete PR please 😄 

Also, we will have to update the documentation